includes/class-wc-gateway-amazon-payments-advanced-subscriptions.php

```diff
@@ -405,9 +420,14 @@ public function copy_meta_from_sub( $meta, $order, $subscription ) {
 		);
 
 		foreach ( $meta_keys_to_copy as $key ) {
+			$value = $subscription->get_meta( $key );
+			if ( empty( $value ) ) {
```

Comment:
we are sure that any meta to copy comes with 0 as value here?

Reply:
yes, none of this would be 0 at any point